src/main.ts

```diff
@@ -221,6 +222,9 @@ export default class LinterPlugin extends Plugin {
     if (Object.prototype.hasOwnProperty.call(storedSettings, 'lintCommands')) {
       this.settings.lintCommands = storedSettings.lintCommands;
     }
+    if (Object.prototype.hasOwnProperty.call(storedSettings, 'customRegexs')) {
+      this.settings.customRegexs = storedSettings.customRegexs;
+    }
```

Comment:
This may need updating to the new form of settings that just got pushed up, but if not, I can go ahead and update it after the fact.

Reply:
Hey, I am confused where is the new form of settings

Reply:
Here is the new format they follow: https://marcus.se.net/obsidian-plugin-docs/user-interface/settings

It uses default settings in main: https://github.com/platers/obsidian-linter/blob/master/src/main.ts#L43-L58